docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,111 @@
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, must users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **Peer TLS certificate and private key**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `tls.rootcert.file` parameter in `core.yaml`.
+        - Copy the TLS CA root certificate `ca-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca-cert.pem`. The path and name of the certificate corresponds to `tls.clientRootCAs.files` parameter in `core.yaml`.
+        - After you have generated the peer TLS certificate, rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
+        - Copy the peer TLS certificate and private key to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls`. The path and name of these certificates correspond to the `tls.cert.file` and `tls.key.file` parameters in the `core.yaml`.
+      - **Peer organization admin MSP**:
+        - After you have registered the peer organization admin user and enrolled the identity, copy the generated MSP folder to `organizations/peerOrganizations/org1.example.com/msp`. This is the peer organization MSP definition and is used by every peer node in the organization. This path becomes the value of the `mspConfigPath` parameter in the `core.yaml` file.
+        - Fabric introduced the concept of "Node Organization Unit (OU)" support that allows a role to be conferred onto an identity by setting an OU value inside a certificate. [Node OU](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) support is enabled by creating a `config.yaml` file inside the `/msp` folder in the `organizations/peerOrganizations/org1.example.com` path in which a role is tied to the root of trust for the issued certificate. **Important:** After you generate the file, remember to edit it and set the path and file name of the root CA certificate for each role. The name of the root CA certificate file is included in the generated MSP under `/msp/cacerts`).
+      - **Peer enrollment certificate and private key**:
+        - After you have registered the peer user for this node and enrolled its identity with the peer organization CA, copy the generated `msp` folder and replace the **local MSP** with it under `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp`.
```

Comment:
Again, don't use the test network location.
The local msp goes in the peer.mspConfigPath location, by default that is /msp.